docs/api/resources/abuse-reports.mdx

```diff
@@ -75,7 +75,7 @@ Returns a 200 response on success
 | `urls` | List&lt;string&gt; | a list of URLs containing suspected abusive content |
 | `metadata` | string | arbitrary user-defined data about this abuse report. Optional, max 4096 bytes. |
 | `status` | string | Indicates whether ngrok has processed the abuse report. one of `PENDING`, `PROCESSED`, or `PARTIALLY_PROCESSED` |
-| `hostnames` | [AbuseReportHostname](#abusereporthostname-fields-1) | an array of hostname statuses related to the report |
+| `hostnames` | [AbuseReportHostname](#abusereporthostname-fields) | an array of hostname statuses related to the report |
```

Comment:
TODO: a lot of these `-1` links get trimmed with the auto generation. The good news is that with the edits to the gen process, there will be _zero_ broken links in the API docs. The bad news is they always go to the first table. **Would you rather leave these manually edited, or wait until this can be permanently fixed?** This is a non-trivial problem to fix, to know where to add the `-1`, '-2', etc